db/migrate/20200212205137_add_institution_id_to_versioned_school_certifying_officials_archives.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstitutionIdToVersionedSchoolCertifyingOfficialsArchives < ActiveRecord::Migration[5.2]
+  def change
+    add_column :versioned_school_certifying_officials_archives, :institution_id, :bigint
+  end
+end
```

Comment:
This is unnecessary.  The previous migration (`20200212204337_add_institution_id_to_versioned_school_certifying_officials.rb `) takes care of this.

Reply:
I think that we actually do need this migration, since `versioned_school_certifying_officials` and `versioned_school_certifying_officials_archives` are both created before `institution_id` is added to `versioned_school_certifying_officials`, `institution_id` will not be automatically added to `versioned_school_certifying_officials_archives`.

Reply:
Ooooops, I totally misread the table name.  Sorry!

Reply:
Not a problem! With the relatively long and similar names, it is easy to do so.